.github/workflows/builds.yml

```diff
@@ -62,28 +69,28 @@ jobs:
         BUILD_TYPE: ["Release", "Debug"]
         PLATFORM:
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: --preset eigen -DALGEBRA_PLUGINS_USE_SYSTEM_EIGEN3=TRUE
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: --preset eigen
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: -DALGEBRA_PLUGINS_INCLUDE_SMATRIX=TRUE
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: --preset vc
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: --preset vecmem
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:56"
             OPTIONS: --preset fastor
           - NAME: "CUDA"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_cuda:67"
             OPTIONS: --preset cuda
           - NAME: "SYCL"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_oneapi:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404_oneapi:56"
```

Comment:
Why not use the same (`67`) tag for all of these?

And while at it, could you add SYCL builds with `"ghcr.io/acts-project/ubuntu2404_cuda_oneapi:67"` and `"ghcr.io/acts-project/ubuntu2404_rocm_oneapi:67"` as well? I'd be curious to see if those succeed... 🤔

Reply:
Updated all to 68

Reply:
Went back to 56 as some additional ROOT config would be needed with the latest containers